doc/api/fs.markdown

```diff
@@ -340,6 +340,12 @@ The file is created if it does not exist.
 
 * `'ax+'` - Like `'a+'` but fails if `path` exists.
 
+On POSIX platforms `flags` can also be a number as documented by open(2);
```

Comment:
Not just POSIX, it works on Windows too.  Not all flags from `require('constants')` are available but the common ones are.